airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/InitialSyncCtidIterator.java

```diff
@@ -79,7 +88,9 @@ public InitialSyncCtidIterator(final CtidStateManager ctidStateManager,
                                  final int maxTuple,
                                  final FileNodeHandler fileNodeHandler,
                                  final boolean tidRangeScanCapableDBServer,
-                                 final boolean useTestPageSize) {
+                                 final boolean useTestPageSize,
+                                 final Instant startInstant,
+                                 final Optional<Duration> cdcInitialLoadTimeout) {
```

Comment:
supernit and optional: slightly fancier way: To pass in expiration instant here and compare that with Instant.now()

Reply:
Ack, will just keep as is to keep parity with others